### ISSUE TITLE
Using phantom-pool and phantom modules can make exporting more easy

### ISSUE THREAD

**Issue by popo1221** [highcharts/node-export-server#101]
Hi all,

I think using phantom-pool and phantom modules can make exporting more easy. 

```js
exports.createWorker = function createWorker(options) {
    return async function render(instance) {

        const page = await instance.createPage();
        const status = await page.open(getExporterPath());
        if (status !== 'success') {
          throw new Error('Cannot open exporter.html');
        }
        
        await page.evaluate(function renderChart(data, template) {
            // Code here to render chart
            window.renderHighCharts(options);
        }, data, template);
        
        const base64 = await page.renderBase64('png');
        return base64;
    }
};
```

Will you guys use them in the future?

**Comment by crapthings** [comment]
headless is better

**Comment by popo1221** [comment]
Hi @crapthings. I am interested in your proposal, can you give some examples about headless?
I have tested Puppeteer. But its page. screenshot is a little slow.

**Comment by cvasseng** [comment]
@popo1221 We looked into phantom-pool when starting this project, and it did not perform very well for our use case (it was very, very slow when dealing with large numbers of exports), which is why we built our own pooling on top of `phantomjs-prebuilt`.

Regardless, as @crapthings said, headless is better, and we will be switching to that in the future. We don't have an ETA at the moment, as we're still in the early phase of investigating the best approach, and coming up with a plan to make the transition as smooth as possible.
